### PR TITLE
[Fixes #157] Flask app for model deployment

### LINKED ISSUE #157: Flask app for deployment
## Feature Request

Is your feature request related to an existing issue?
No

### Feature Description
Create a simple flask app that uses a given model. 

### Additional context
This will be used for deploying the created models.

### PR DESCRIPTION
**Pull request by Onkar302**
## Pull Request

fixes #157 

### Description
* Added Flask app for model deployment with folder name flaskv1 inside v1 folder.

### Checklist:
- [x] Followed the guidelines mentioned in our [Contributing document](../CONTRIBUTING.md)
- [x] Ensured that there aren't other open [Pull Requests](https://www.github.com/Auto-DL/Generator/pulls) for the same update/change
- [x] This PR is linked to an [Issue](https://www.github.com/Auto-DL/Generator/issues) and the issue number has been mentioned in the title and body of the PR
- [x] The code is clean and docstrings have been added or updated as required
- [x] Code is linted/formatted locally prior to submission (using `black`)

Signed-off-by: Onkar302 onkarbhatlawande@gmail.com

### CODE REVIEW

**Review by ADI10HERO — CHANGES_REQUESTED** [review]
Please make the discussed changes.

**Comment by Onkar302** [comment]
> Please make the discussed changes.

Implemented the changes kindly have a look.

File: BackEndApp/v1/flaskv1/templates/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	<h1>Classification Model</h1>
+
+	<form action='/predict' method="post" enctype="multipart/form-data">
+  		<label for="img">Select image:</label>
+  		<input type="file" id="img" name="img">
+  		<input type="submit">
+	</form>
+
+</body>
+</html>
```
Comment (shintan777) [inline BackEndApp/v1/flaskv1/templates/index.html:18]:
```suggestion
</html>

```

File: BackEndApp/v1/flaskv1/templates/result.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+	<title>Flask App</title>
+</head>
+<body>
+	
+	<h1>Result</h1>
+
+	<p>Its a {{prediction}}</p>
+
+</body>
+</html>
```
Comment (shintan777) [inline BackEndApp/v1/flaskv1/templates/result.html:14]:
```suggestion
</html>

```

**Review by shintan777 — CHANGES_REQUESTED** [review]

**Comment by Onkar302** [comment]
Does that mean adding an empty line at the end of both the HTML files?

**Comment by Onkar302** [comment]
Done, please have a look.

**Pull request closed without merging**